packages/suite/src/components/suite/QuestionTooltip/index.tsx

```diff
@@ -17,6 +17,7 @@ interface Props {
     label?: JSX.Element | ExtendedMessageDescriptor['id'];
     tooltip?: JSX.Element | ExtendedMessageDescriptor['id'];
     size?: number;
+    className?: string;
```

Comment:
unused prop

Reply:
It is used for styled components.

Reply:
could you point me to implementation then? because i couldn't find any